### DISCUSSION TITLE
Run queries/mutations from the /server/api directory

### DISCUSSION THREAD

**Discussion by imduchy** [nuxt-community/apollo-module#543] [Q&A]
Hello,

is it possible to run queries & mutations from the `/server/api` directory? I've tried the following but it seems that composables are not available here.

```ts
// server/api/users.get.ts

import { getUser } from "../graphql/queries";

export default eventHandler(async () => {
  const { data } = await useQuery(getUser, { username: "alicia" });

  return data;
});
```

My GraphQL server is running on AWS AppSync and I'm trying to make requests against it from the server side (`/server/api`). What would be the correct approach to this?

**Comment by alancwoo** [comment]
I have the same question which seems to also have been asked here with no solution: https://github.com/nuxt-modules/apollo/discussions/459

Definitely need to figure out a solution to this/querying graphql from the server-side

**Comment by martinszeltins** [comment]
I am looking for the exact same thing!
